homeassistant/components/aussie_broadband/sensor.py

```diff
@@ -143,7 +143,13 @@ def __init__(
     def native_value(self):
         """Return the state of the sensor."""
         if self.entity_description.key == "internet":
-            return self.coordinator.data[self.entity_description.key]["kbytes"]
+            try:
+                return self.coordinator.data[self.entity_description.key]["kbytes"]
+            except KeyError:
+                return None
         if self.entity_description.key in ("national", "mobile", "sms"):
-            return self.coordinator.data[self.entity_description.key]["calls"]
+            try:
+                return self.coordinator.data[self.entity_description.key]["calls"]
+            except KeyError:
+                return None
```

Comment:
I think it would be cleaner to use get:
```suggestion
            return self.coordinator.data[self.entity_description.key].get("calls")
```

Reply:
Cant believe I forgot about .get(), thanks!